src/Common/src/TypeSystem/IL/Stubs/UnsafeIntrinsics.cs

```diff
@@ -49,6 +52,12 @@ public static MethodIL EmitIL(MethodDesc method)
                         (byte)ILOpcode.ldarg_0, (byte)ILOpcode.ldarg_1,
                         (byte)ILOpcode.prefix1, unchecked((byte)ILOpcode.ceq),
                         (byte)ILOpcode.ret }, Array.Empty<LocalVariableDefinition>(), null);
+                case "ByteOffset":
+                    return new ILStubMethodIL(method, new byte[]
+                    {
+                        (byte)ILOpcode.ldarg_0, (byte)ILOpcode.ldarg_1,
```

Comment:
Should this the ldarg order be reversed here?
`(byte)ILOpcode.ldarg_1, (byte)ILOpcode.ldarg_0,`

Reply:
Yes. Good catch